### DISCUSSION TITLE
How is `MotifSimilarity` column in `motifAnnotations_mgi` generated ?

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [aertslab/scenicplus#464] [Q&A]
I collected motifs for my niche model organism (without direct annotation) only by the **`TF orthology relationship`.** But it seems not enough.

I wonder how does `Scenic` infer motifs by its **similarity** .
Thank you for your help!

**Comment by SeppeDeWinter** [comment]
Hi @jiangpuxuan 

What do you mean by
>  [...] But it seems not enough.

You can for sure use SCENIC+ with only motif annotations based on orthology.

> I wonder how does Scenic infer motifs by its similarity .

This is done using TomTom, to calculate the similarity to other annotated motifs.

All the best,

Seppe
